local-tests/chainrunner/chain.py

```diff
@@ -169,14 +169,26 @@ def fork(self, forkoff_path, ws_endpoint, snapshot_file=None):
         chainspec = join(self.path, 'chainspec.json')
         snapshot = snapshot_file if snapshot_file else join(self.path, 'snapshot.json')
         cmd = [check_file(forkoff_path), '--ws-rpc-endpoint', ws_endpoint,
-                '--initial-spec-path', chainspec,
-                '--snapshot-path', snapshot,
-                '--combined-spec-path', forked]
+               '--initial-spec-path', chainspec,
+               '--snapshot-path', snapshot,
+               '--combined-spec-path', forked]
         if snapshot_file:
             cmd.append('--use-snapshot-file')
         subprocess.run(cmd, check=True)
         self.set_chainspec(forked)
 
+    def get_highest_imported(self, nodes=None):
+        """Return the highest block id imported by all the selected nodes"""
```

Comment:
let's not say `id` if it's only height - two nodes may import different blocks of the same height, and this function will still return something